misc/tests/conftest.py

```diff
@@ -0,0 +1,8 @@
+def pytest_addoption(parser):
```

Comment:
I think this new file requires some documentation: why do we need it and what does it do? The name `conftest.py` is not that self-explanatory.

Reply:
I also didn't see where this option is ever used.